scripts/runTests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Need this script instead of simply running `npm run test:okta-hosted-login && npm run test:custom-login` because - 
* I've added a "hack" in TCK that will kill the `node.exe` process after any `npm` script exits. E.g - `npm run test:okta-hosted-login` will start `npm run okta-hosted-login-server`. After the 1st script exits, I kill all `node.exe` processes which in turn kills the 1st script also
* Due to this "hack" (found no way around in windows to just kill the child process), only the 1st npm script in `npm run test:okta-hosted-login && npm run test:custom-login` will run.
* To run both the tests/scripts, I have to run them one after another so that no unwarranted killing happens 
* I know it's messy but I spent enough time to find a better way but couldn't come up with a better solution